src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
It seems you are linking a SCSS file in your HTML. Browsers can't interpret SCSS, they only understand CSS. You need to compile your SCSS to CSS and link the resulting CSS file in your HTML.